bin/openssh-ldap-publickey

```diff
@@ -120,16 +131,16 @@ sub parse_nss_base_passwd {
     my $nss_base_passwd = shift;
 
     my $result = {
-        'nss_base_passwd' => undef,
-        'scope'           => undef,
+        'base_passwd' => undef,
+        'scope'       => undef,
     };
 
     # Support nss_base_passwd with scope notation
     # example:
     # nss_base_passwd ou=People?one
```

Comment:
should the comment and the methods names be updated too?

Reply:
I don't think so. This code is still specific to parsing `nss_base_passwd` only. Just the name of the internal variable it populates has changed.